assets/mail-templates/failure.txt

```diff
@@ -12,6 +12,6 @@ If you think you found a bug, please fill in and submit this issue:
 
 ~ @TIMESTAMP
 
--- 
+--
```

Comment:
I think `--` (plus a trailing space) is kind of a standard ([link](https://en.wikipedia.org/wiki/Signature_block#Signatures_in_Usenet_postings), [link](https://together.jolla.com/question/2884/email-signature-separator-use-standard-dash-dash-space-/)).
No sure if some MUA out there will *not* parse this signature correctly if it doesn't have that space&hellip;?
(Same comment about the other template.)

Reply:
ah, sorry, my editor is configured to remove the trailing whitespaces automatically.